inventory/upcloud.py

```diff
@@ -116,37 +123,39 @@ def assign_ips_to_servers(servers):
         object.__setattr__(server, 'ip_addresses', [])
 
     # assign IPs to their corresponding server
-    ips = manager.get_ips()
+    ips = manager.get_ips(ignore_ips_without_server=True)
```

Comment:
This is regarding a problem in the actual api that I mentioned in slack, some ips return without server so we have to ignore them.